pkg/device/sysfs_linux.go

```diff
@@ -106,3 +108,49 @@ func getHolders(name string) ([]string, error) {
 func getDMName(name string) (string, error) {
 	return readFirstLine("/sys/class/block/" + name + "/dm/name")
 }
+
+// GetStat returns statistics for a given device name.
+func GetStat(name string) (stats []uint64, err error) {
```

Comment:
can we define

```go
type Stats struct {
	ReadSectorBytes  float64
	ReadTicks        float64
	WriteSectorBytes float64
	WriteTicks       float64
	TimeInQueue      float64
}
```

in this pkg and return (Stats, error) here?

Reply:
If we want to return a structure, it should contain all the values not selected values. It is better to leave the logic in the higher layer